### PR TITLE
WINDUPRULE-422 @ContextName and @ContextNames have been removed

### PR DESCRIPTION
**Pull request by johnpoth**
https://issues.redhat.com/browse/WINDUPRULE-422

Thanks !

cc @mrizzi

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by mrizzi** [comment]
test this please

**Comment by johnpoth** [comment]
@mrizzi do you know when this could be merged ? Thanks !

File: rules-reviewed/camel3/camel2/java-multiple-camelcontexts-per-application-not-supported.windup.xml [hunk 1]
```diff
@@ -32,5 +32,23 @@
...
             </where>
         </rule>
+        <rule id="java-multiple-camelcontexts-per-application-not-supported-00001">
+            <when>
+                <javaclass references="org.apache.camel.cdi.{CdiCamelContextName}">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="{CdiCamelContextName} has been removed" effort="2" category-id="mandatory">
```
Comment (mrizzi) [inline rules-reviewed/camel3/camel2/java-multiple-camelcontexts-per-application-not-supported.windup.xml]: For efforts (i forgot to tell you, sorry) we use some fixed values for which we provide documentation [How Story Points are Estimated in Rules](https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/4.3/html-single/rules_development_guide/index#how_story_points_are_estimated_in_rules).
Hence i proposed to change the effort to `3`.
```suggestion
                <hint title="{CdiCamelContextName} has been removed" effort="3" category-id="mandatory">
```

**Comment by johnpoth** [comment]
@mrizzi those changes are fine with me! Thanks for reviewing! I'll update my code with the link you sent me regarding story points

**Comment by mrizzi** [comment]
retest

**Comment by johnpoth** [comment]
Thanks @mrizzi !

**Pull request merged**